bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -1,195 +1,404 @@
-| returntemp                | Number:Temperature   | Return water temperature                                 | R      |
-| outsidetemp               | Number:Temperature   | Outside temperature                                      | R/W    |
-| waterpressure             | Number:Pressure      | Central heating water pressure                           | R      |
-| ch_enable                 | Switch               | Central heating enabled set at boiler                    | R      |
-| ch_enablerequested        | Switch               | Central heating enabled requested by thermostat          | R      |
-| ch_enableoverride         | Switch               | Central heating enabled overridden at gateway            | R      |
-| ch2_enable                | Switch               | Central heating 2 enabled set at boiler                  | R      |
-| ch2_enablerequested       | Switch               | Central heating 2 enabled requested by thermostat        | R      |
-| ch2_enableoverride        | Switch               | Central heating 2 enabled overridden at gateway          | R      |
-| ch_mode                   | Switch               | Central heating active                                   | R      |
-| dhw_enable                | Switch               | Domestic hot water enabled                               | R      |
-| dhw_mode                  | Switch               | Domestic hot water active                                | R      |
-| flame                     | Switch               | Burner active                                            | R      |
-| modulevel                 | Number:Dimensionless | Relative modulation level                                | R      |
-| maxrelmdulevel            | Number:Dimensionless | Maximum relative modulation level                        | R      |
-| fault                     | Switch               | Fault indication                                         | R      |
-| servicerequest            | Switch               | Service required                                         | R      |
-| lockout-reset             | Switch               | Lockout-reset enabled                                    | R      |
-| lowwaterpress             | Switch               | Low water pressure fault                                 | R      |
-| gasflamefault             | Switch               | Gas or flame fault                                       | R      |
-| airpressfault             | Switch               | Air pressure fault                                       | R      |
-| waterovtemp               | Switch               | Water over-temperature fault                             | R      |
-| oemfaultcode              | Switch               | OEM fault code                                           | R      |
-| diag                      | Switch               | Diagnostics indication                                   | R      |
```

Comment:
@ArjenKorevaar one more question: last year you introduced this `diag` channel which (I think) would indicate that the OTGW is talking to some kind of master or slave device (??). It is not that well documented. But I am surprised that you deleted it. If my assumption above is true then it would probably be a channel of the `openthermgateway` bridge thing, rather than a channel on a `boiler` or `ventilationheatrecovery` thing. ??

Reply:
I think removing the diag channel was done unintentionally. Since the diag code is sent by the actual unit rather than the OTGW device (its part of the OpenTherm specifications), the channel should be on the boiler thing. I don't know if ventilation / heatrecovery units in general make use of this message id as well. And even then, some types of units may support it, others may not. I would usually add them on request.

Reply:
Apparently `diag` wasn't deleted from DataItemGroup, so presumably it is still supported in reality? In that case it just needs to be restored to the ReadMe. Or??

Reply:
.. and it also needs to be restored the `boiler.xml` file ..

Reply:
..see this https://github.com/openhab/openhab-addons/pull/11203/commits/47d4696e62a9a98d0cad53b477b5bab3f3286e76

Reply:
so resolved

Reply:
Indeed, the DataItemGroup is basically generated from the list of all possible messages id's of the OpenTherm specifications. The only thing that is needed then is a channel to map the incoming value to.